backend/twitterbot/models/tweeted.py

```diff
@@ -12,13 +12,13 @@
 ]
 
 
-class TweetedVideo(models.Model):
+class TweetInfo(models.Model):
     """One tweeted video."""
 
     video = models.ForeignKey(
         Entity,
         on_delete=models.CASCADE,
-        related_name="video",
+        related_name="tweet_info",
```

Comment:
"video" was indeed misleading. However, this `related_name` represents a queryset for all "tweet info" instances that are related to this video. 

So `tweet_info_set` may be more appropriate, or simply `tweets` ?

Reply:
Good catch. Renamed to `tweets` in 42162947